routes/api/actions-router.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+
+const Actions = require("../../data/helpers/actionModel");
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const actions = await Actions.get();
+    res.status(200).json(actions);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      msg: "cannot find projects"
+    });
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const actions = await Actions.get(req.params.id);
+    res.status(200).json(actions);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      msg: "cannot find projects"
+    });
+  }
+});
+
+router.post("/", async (req, res) => {
+  try {
+    const actions = await Actions.insert(req.body);

```

Comment:
Good job, ideally we would have some error handling here for is a project id doesn't exist or if required fields aren't filled out